src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useNavigate } from 'react-router-dom';
+
+import Header from 'components/Header';
+import { buttonVariant } from 'components/ui/Button';
+import PageHeading from 'components/ui/PageHeading';
+import PageText from 'components/ui/PageText';
+
+import styles from './NotFoundPage.module.css';
+
+const NotFoundPage = () => {
+  const navigate = useNavigate();
+
+  const handleGoBack = () => {
+    navigate(-1);
+  };
+
+  return (
+    <>
+      <Header />
+      <div className={styles.notFoundPage}>
+        <div className={styles.pageContainer}>
+          <div className={styles.notFoundPageWrapper}>
+            <div className={styles.bgImage} />
+            <div className={styles.content}>
+              <PageHeading className={styles.title}>
+                Check your navigation!
+              </PageHeading>
+              <div className={styles.text}>
+                <PageText className={styles.textContent}>Error: 404</PageText>
+                <PageText className={styles.textContent}>
+                  We're very sorry. This page wasn't found. Check your map and
+                  find the right way!
```

Comment:
I think you have to get this message as a props for a component, and if that absents you show default massage